db/seeders/letter_template_seeder.rb

```diff
@@ -101,7 +101,7 @@ def seed!
                   <br><br>Dear Colleague
                   <br>
                   <br><strong>DATA PROTECTION ACT 2018: SUBJECT ACCESS REQUEST</strong>
-                  <br><strong><%= values.subject_full_name&.upcase %> - <%= values.first_prison_number %></strong>
+                  <br><strong><%= values.subject_full_name&.upcase %><% if values.prison_number.present? %> - <%= values.first_prison_number %></strong><% end %>
```

Comment:
Should the end be before the </strong> ?

Reply:
@digitalali-moj ,  no,  it is syntax basically it means the prison-number with hyphen will be displayed only if the case has the prison_number

Reply:
@digitalali-moj ,  thank you for spotting this,  corrected them